72Afes/PagerTab/Event/EventCell.swift

```diff
@@ -8,12 +8,18 @@
 
 import UIKit
 import UserNotifications
+import NativePopup
 
 class EventCell: UICollectionViewCell {
     
     // 終わりの時間過ぎたら暗くしたいけど、詳細見れるようにしたいねえ
     // まあそもそもこのUI気に入ってないから変えるとは思うんだけど
     
+    var status: eventStatus = .finished
+    enum eventStatus {
```

Comment:
```suggestion
    enum EventStatus {
```

enumはクラスとかと同じなので大文字だとよき